UndertaleModTool/Editors/UndertaleTexturePageItemEditor.xaml.cs

```diff
@@ -33,6 +36,13 @@ private void Import_Click(object sender, RoutedEventArgs e)
                 Bitmap image = TextureWorker.ReadImageFromFile(dlg.FileName);
                 image.SetResolution(96.0F, 96.0F);
                 (this.DataContext as UndertaleTexturePageItem).ReplaceTexture(image);
+
+                // Refresh the image of "ItemDisplay"
+                if (ItemDisplay.FindName("RenderAreaBorder") is not Border border)
+                    return;
+                if (border.Background is not ImageBrush brush)
+                    return;
```

Comment:
Variable names are confusing here. If they pass the check, they *arent* borders/brushes, yet you keep calling them like that.